### PR TITLE
Updated the repo Readme file

### LINKED ISSUE #161: Docs: Improving the repo Readme
### Description

# Problem Description: 
The current README lacks some essential features that can enhance user understanding and project visibility

# Proposed Changes/Solution
- Currently, the readme lacks some features such as: 
   - Badges to showcase project status, license, version, and other relevant information. Badges provide quick insights into the project's health and status.   
   - Include screenshots or GIFs showcasing different parts of the application. Visuals provide users with a quick overview of the app's interface and features.
   - Any other information about the application including the tests, guidelines, security, permissions explanation, resource usage, etc.

Please consider assigning this issue to me if you find it to be acceptable.

### PR DESCRIPTION
**Pull request by its-me-abhishek**
### Description
- Added GSoC image for redirecting to the project.
- Added screenshots of the views.
- Altered the future plans that have been implemented (custom ringtone).

## Fixes #161
Link for readme: 
https://github.com/its-me-abhishek/ultimate_alarm_clock/blob/main/README.md
## Checklist

- [x] Tests have been added or updated to cover the changes
- [x] Documentation has been updated to reflect the changes
- [x] Code follows the established coding style guidelines
- [x] All tests are passing

### CODE REVIEW

**Comment by its-me-abhishek** [comment]
Hi @MarkisDev, I've made some tweaks to the readme file. Please review my PR.

File: README.md [hunk 1]
```diff
@@ -227,9 +226,13 @@ The "Ultimate Alarm Clock" offers a user-friendly and versatile interface design
 
 ### Splash Screen
 - The app opens with a welcoming splash screen, providing a brief introduction to the application.
+<img src="https://github.com/its-me-abhishek/ultimate_alarm_clock/assets/114338679/ae0ad406-6b6a-432f-8fec-e363bc0aa14c" height="400" alt="splash">
```
Comment (MarkisDev) [inline README.md:229]: We should definitely move to relative links

**Review by MarkisDev — CHANGES_REQUESTED** [review]
Please use relative links wherever possible!

**Comment by its-me-abhishek** [comment]
I've made a folder in the assets/images to store  all the images and have updated all the links to relative links.

**Pull request merged**